KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/NWCommands.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) CoSMo Software Consulting Pte. Ltd. - All Rights Reserved
+ */
+
+package io.cosmosoftware.kite.instrumentation;
+
+import io.cosmosoftware.kite.exception.KiteTestException;
+import io.cosmosoftware.kite.report.Status;
+
+import javax.json.JsonObject;
+
+public class NWCommands {
+
+  private final String inter;
+  private final String gateway;
+  private final String finalCommand;
+  private final int delay;
+  private final int packetloss;
+  private final int corrupt;
+  private final int duplicate;
+  private final JsonObject bandwidth;
+
+  public NWCommands(JsonObject jsonObject, Instrumentation instrumentation) throws Exception {
+    try {
+      this.gateway = jsonObject.getString("gateway");
+      if (instrumentation.get(gateway) == null) {
+        throw new Exception(" Error in json config scenario, gateway specified is not in the instrumentation file ! ");
+      }
+    } catch (NullPointerException e) {
+      throw new KiteTestException("Error in json config scenario, the key gateway is missing.", Status.BROKEN, e);
+    }
+    this.inter = jsonObject.containsKey("interface") && !jsonObject.containsKey("command") ? jsonObject.getString("interface") : null;
```

Comment:
only one interface?

Reply:
The way I see it a scenario is like this : 

{
            "clientId": 0,
            "name": "Scenario 1",
            "duration": 10000,
            "commandList": [
              {
                "gateway": "gw1",
                "interface": "enp0s8",
                "delay": 50,
               "corrupt": 5
              },
              {
                "gateway": "gw1",
                "interface": "enp0s9",
                "packetloss": 10
              }
            ]
          }

Maybe it needs some change but for now CommandList has only one interface by command

Reply:
bw needs to be done on both interfaces

Reply:
I know and for now the client needs to specify he wants to limit bandwidth on both interfaces

Reply:
I feel like we need to change Instance to make it easier, for now interfaces are specified like this : 

{
      "_id": "gw1",
	  "type": "gateway",
      "name": "GW1",
      "ipAddress": "192.168.2.101:14501",
      "username": "osboxes",
      "password": "osboxes.org",
      "keyFilePath": "~/.ssh/osboxes",
	  "nit0": "enp0s3",
	  "nit1": "enp0s8",
	  "nit2": "enp0s9"
    }

I think if we say nit is a list of interface it is more flexible and easier to use than nit0, nit1 and nit2. What do you think ?

Reply:
Cause I feel like making nit a list make things more complicated for no real benefit, instead we could make nit optional for bandwidth command and then we run the command on both interfaces without the client specifying anything. We could do the same for other commands since the interface the command is ran on doesn't really matters in the end.